composites/OnboardingWizard/OnboardingWizard.js

```diff
@@ -259,6 +276,42 @@ class OnboardingWizard extends React.Component {
 		return ( hideButton ) ? "" : <RaisedButton className={ className } { ...attributes } />;
 	}
 
+	/**
+	 * Updates the currentStepId in the state when the hash in the URL changes.
+	 *
+	 * @returns {void}
+	 */
+	listenToHashChange() {
+		// Because the hash starts with a hashtag, we need to do `.substring( 1 )`.
+		this.setState( { currentStepId: window.location.hash.substring( 1 ) } );
+	}
+
+	/**
+	 * When the currentStepId in the state changes, push the new hash to the history if differs from the current hash.
+	 *
+	 * If we wouldn't check against the current hash, it would lead to double hashes when using the browser's previous
+	 * and next buttons.
+	 *
+	 * @param {Object} prevProps The previous props.
+	 * @param {Object} prevState The previous state.
+	 *
+	 * @returns {void}
+	 */
+	getSnapshotBeforeUpdate( prevProps, prevState ) {
```

Comment:
What is the usecase for this function, it isn't used?

Reply:
This runs before componentDidMount, and is one of the React lifecycle methods.